gcloud/storage/test_bucket.py

```diff
@@ -674,75 +674,57 @@ def test_id(self):
         bucket = self._makeOne(properties=properties)
         self.assertEqual(bucket.id, ID)
 
-    def test_get_lifecycle_eager(self):
+    def test_location_getter(self):
```

Comment:
Very strange diff choice by `git` / GitHub. This could have easily been seen as a deletion of the `test_*_lifecycle_*` tests and an addition of the others.